configs/ecospheres/config.yaml

```diff
@@ -29,7 +29,7 @@ website:
   # leave empty if not pertinent
   logo_operator:
   homepage:
-    title: 'La plateforme des données de la transition écologique'
+    title: 'Le&nbsp;catalogue des&nbsp;données&nbsp;de<br /> la <span class="highlight">transition écologique</span>'
```

Comment:
Ne pas utiliser la conf, vu la tête de la chaine et le fait qu'on a une homepage custom, ça n'a pas d'intérêt je pense. Donc laisser le title vide ici (cf leave empty if not pertinent) et mettre le HTML directement dans le template.

Reply:
Désolé, à part la couleur du titre, le reste a sauté avec le rebase ou une gestion de conflit.

Reply:
Concernant ta remarque sur le placeholder, pas sur d'avoir compris ce que tu appelles hint. En tous cas le comportement était buggé post rebase, mon dernier commit rétablit le comportement comme proposé initialement. Pas de placeholder en haut, le même sur le grand, focus ou pas dessus (le layout utilise 2 éléments html pour le même placeholder)

Reply:
> Concernant ta remarque sur le placeholder, pas sur d'avoir compris ce que tu appelles hint. En tous cas le comportement était buggé post rebase, mon dernier commit rétablit le comportement comme proposé initialement. Pas de placeholder en haut, le même sur le grand, focus ou pas dessus (le layout utilise 2 éléments html pour le même placeholder)

Ok, c'est plus clair ! Merci.